app/models/payment/go_cardless/transaction.rb

```diff
@@ -81,8 +81,10 @@ def publish_failed_subscription_charge
     ChampaignQueue.push({
       type: 'subscription-payment',
       params: {
+        created_at: created_at,
         recurring_id: subscription.go_cardless_id,
-        success: 0
+        success: 0,
+        status: 'failed'
```

Comment:
Yep - I was only passing `success` for a while, which isn't the field that Simona queries, so `status` also needs to be passed, but I also want to keep `success` in for consistency (it's used in BT subscription charge failures also).